src/email/aws.email.service.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  SESClient,
+  SendEmailCommand,
+} from "@aws-sdk/client-ses";
+import { Email } from "./dto/email.dto";
+import { BaseEmailService } from "./interfaces/email.service.interface";
+import config from "./../utils/config";
+import { InjectMapper } from "@automapper/nestjs";
+import { Mapper } from "@automapper/core";
+
+export class AwsEmailService extends BaseEmailService {
```

Comment:
"aws-email.service" is a better naming for the file. Also, we can move this service under email/providers folder.